src/writer/vault.py

```
@@ -0,0 +1,53 @@
+import os
+from typing import Dict, Union
+import logging
+import requests
+
+
+class WriterVault:
+    def __init__(self) -> None:
+        self.secrets: Union[Dict, None] = None
+
+    def get_secrets(self) -> Dict:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)